recipe/meta.yaml

```diff
@@ -16,52 +15,61 @@ outputs:
   - name: oxigraph-server
     build:
       script:
-        - bash {{ RECIPE_DIR }}/build-oxigraph.sh  # [unix]
+        - bash {{ RECIPE_DIR }}/build-oxigraph.sh        # [unix]
         - call {{ RECIPE_DIR }}\bld-oxigraph-server.bat  # [win]
     requirements:
       build:
-        - {{ compiler('c') }}                    # [unix]
-        - {{ compiler('cxx') }}                  # [unix]
-        - {{ compiler('m2w64_c') }}              # [win]
-        - {{ compiler('m2w64_cxx') }}            # [win]
+        - {{ compiler("c") }}                    # [unix]
+        - {{ stdlib("c") }}                      # [unix]
+        - {{ compiler("cxx") }}                  # [unix]
+        - {{ compiler("m2w64_c") }}              # [win]
+        - {{ stdlib("m2w64_c") }}                # [win]
+        - {{ compiler("m2w64_cxx") }}            # [win]
         - m2w64-binutils                         # [win]
-        - {{ min_rust }}
-        - cargo-bundle-licenses
         - clangdev                               # [not osx]
         - llvmdev                                # [not osx]
+        - {{ compiler("rust") }}
+        - cargo-bundle-licenses
+      host:
+        - openssl
       run:
         - ca-certificates
+        - openssl
     test:
       commands:
-        - oxigraph_server --version
-        - oxigraph_server --help
+        - oxigraph --version
+        - oxigraph --help
 
   - name: pyoxigraph
     build:
       script:
-        - bash {{ RECIPE_DIR }}/build-oxigraph.sh  # [unix]
+        - bash {{ RECIPE_DIR }}/build-oxigraph.sh   # [unix]
         - call {{ RECIPE_DIR }}\bld-pyoxigraph.bat  # [win]
     requirements:
       build:
         - python                                 # [build_platform != target_platform]
         - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-        - {{ compiler('c') }}                    # [unix]
-        - {{ compiler('cxx') }}                  # [unix]
-        - {{ compiler('m2w64_c') }}              # [win]
-        - {{ compiler('m2w64_cxx') }}            # [win]
+        - {{ compiler("c") }}                    # [unix]
+        - {{ stdlib("c") }}                      # [unix]
+        - {{ compiler("cxx") }}                  # [unix]
+        - {{ compiler("m2w64_c") }}              # [win]
+        - {{ stdlib("m2w64_c") }}                # [win]
+        - {{ compiler("m2w64_cxx") }}            # [win]
         - m2w64-binutils                         # [win]
-        - {{ min_rust }}
-        - maturin >=1.0,<2.0
-        - cargo-bundle-licenses
         - clangdev                               # [not osx]
         - llvmdev                                # [not osx]
+        - {{ compiler("rust") }}
+        - maturin >=1.0,<2.0
+        - cargo-bundle-licenses
       host:
         - maturin >=1.0,<2.0
+        - openssl
```

Comment:
It should not be required: on Linux pyoxigraph uses rustls, and on Windows/mac, the OS TLS library.